### DISCUSSION TITLE
Create alert from a panel doesn't work with Mimir Alertmanager

### DISCUSSION THREAD

**Discussion by alessandroniciforo** [grafana/grafana#62946] [Alerting]
I'm exploring the integration of Grafana with Mimir Alertmanager and I found a lack of support for creating an alert from a panel.

Steps:
- Open an existing dashboard
- Edit a panel that is using Mimir as a datasource
- Click on the "Alert" tab and then on "Create alert rule from this panel"
- You're brought to the Add rule interface with "Grafana managed alert" pre-selected and the query pre-filled with the one configured in the panel

Actual:
- I change source from "Grafana managed alert" to "Mimir or Loki alert" and I have to manually fill in the query
- The generated alert does not appear in the "Alert" tab of that same panel I created the alert from, although the special labels `__dashboardUid__` and `__panelId__` are correctly pre-filled.

Expected:
- I change source from "Grafana managed alert" to "Mimir or Loki alert" and the query is pre-filled as well
- Once I create the alert, it will appear in the "Alert" tab of that same panel I created the alert from

Regarding the first point, I know it's a minor annoyance since you can copy paste the query before changing selection but it would smoothen the UX to just have it pre-filled. It would also be nice to have an instance-level configuration to select the "default" alert manager so that I can configure Mimir Alertmanager and it would be always pre-selected when creating a new rule (from a panel or not).

For the second point, it seems like everything is in place already to support such a feature. The labels `__dashboardUid__` and `__panelId__` are correctly pre-filled and they can be used to query the alert rules to display for a specific panel.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82932. Feel free to continue the discussion around this there. Thank you!
